Easy_Problems/Coderbyte3.js

```diff
@@ -0,0 +1,11 @@
+function LongestWord(sen) {
```

Comment:
the instructions said if there's 2 words of equal length, print out the first word. Do I leave out that sorting condition?